doc/markdown.rst

```diff
@@ -0,0 +1,44 @@
+.. highlightlang:: python
+
+.. _markdown:
+
+Markdown support
+================
+
+`Markdown <https://daringfireball.net/projects/markdown/>`__ is a lightweight markup language with a simplistic plain
+text formatting syntax.
+It exists in many syntactically different *flavors*.
+To support Markdown-based documentation, Sphinx can use `CommonMark-py <https://github.com/rtfd/CommonMark-py>`__, a
+Python package for parsing the `CommonMark <http://commonmark.org/>`__ flavor. In addition, Sphinx uses
+`recommonmark <http://recommonmark.readthedocs.io/en/latest/index.html>`__, a Docutils bridge to CommonMark.
+
+
+Configuration
+-------------
+
+To configure your Sphinx project for markdown support, proceed as follows:
+
+#. Install CommonMark version **0.5.4** and recommonmark:
```

Comment:
My problem was that I initially installed the packages via `pip install commonmark recommonmark` as suggested in one of the tickets here. `pip install recommonmark` works well, since the requirements are correctly specified in the latest version of recommonmark. I'll adjust the instructions.